tests/actors/rust/http-log-rng-component/src/lib.rs

```diff
@@ -2,8 +2,7 @@ wit_bindgen::generate!("actor");
 
 use serde::Deserialize;
 use serde_json::json;
-use wasmbus_rpc::common::{deserialize, serialize};
-use wasmcloud_actor::*;
+use wasmcloud_actor::{logging, HostRng};
 use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse};
```

Comment:
Nit: Should this test be using the types reexported from `wasmcloud_actor` instead?

Reply:
Yes, eventually, still not sure how to go about http serving in component-model case, I think the most reasonable way to handle this is to wait until `wasi-http` really becomes a thing and then use something that's similar to whatever will happen upstream.
Component model should not be using smithy-generated structs (or at least not directly), this is a temporary solution